Kitodo-API/src/main/java/org/kitodo/api/validation/ValidationResult.java

```diff
@@ -28,7 +28,7 @@
 
     /**
      * Gets valid.
-     * 
+     *
```

Comment:
Change is good but why did your IDE fix this as you did not change anything at this file?

Reply:
Probably because I had the file open